src/GitVersionCore/Core/GitPreparer.cs

```diff
@@ -35,11 +35,9 @@ public void Prepare()
             var currentBranch = ResolveCurrentBranch();
 
             var dotGitDirectory = gitVersionOptions.DotGitDirectory;
-            var projectRoot = gitVersionOptions.ProjectRootDirectory;
```

Comment:
This argument wasn't used, and I removed it whilst refactoring because it helped me.